app/components/project-card-list.cjsx

```diff
@@ -186,7 +186,7 @@ ProjectFilteringInterface = React.createClass
   getDefaultProps: ->
     discipline: ''
     page: 1
-    sort: '-activity'
+    sort: 'display_name'
```

Comment:
Good - better than `-activity` - but see #2424 - not sure this is the best default order. Let's go with this for now though, then if Chris wants something different we can change it under #2424.